website_odoo_debranding/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=================================
+Remove Odoo Branding from Website
+=================================
+
+This module remove Odoo branding on website:
+
+* Remove "Create a free website with Odoo" from footer
+
+Usage
+=====
+
+#. Go to your home page.
+#. Enable or disable *Customize > Footer Copyright > Remove Odoo Promotional
+   Link* at your will. It should be enabled by default.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/11.0
```

Comment:
this is wrong, but see my comment about readme refactoring

Reply:
Sure, Updating it